scripts/scripts.js

```diff
@@ -513,15 +513,25 @@ const centerArticleDivider = (main) => {
   });
 };
 
-const pdfLinkHandler = (link) => {
-  const href = link.getAttribute('href');
-  if (!href) {
-    return;
-  }
-  if (!href.includes('.pdf')) {
-    return;
-  }
-  link.setAttribute('target', '_blank');
+const pdfLinkHandler = () => {
+  const sections = document.querySelectorAll('main > .section');
+
+  const addTargetAttribute = (link) => {
+    const href = link.getAttribute('href');
+    if (!href) {
+      return;
+    }
+    if (!href.includes('.pdf')) {
+      return;
+    }
+    link.setAttribute('target', '_blank');
+  };
+
+  sections.forEach((oElem) => {
```

Comment:
instead of making the querySelectorAll call twice with two forEach loops, you can just do the following:
`const links = document.querySelectorAll('main > .section a');`
and loop through all the links in one shot.

Reply:
@amol-anand code updated. Thanks!